src/site/content/en/learn/pwa/service-workers/index.md

```diff
@@ -1,11 +1,137 @@
 ---
 title: Service workers
 description: >
-  A Service Worker is a crucial component of every PWA. In this chapter, you will learn what it is, the power behind it and how to register one in your code.
+  Service workers are a fundamental part of a PWA and they allow capabilities such as fast loading regardless of the network, offline access, push notifications, and more.
 authors:
   - firt
-date: 2021-11-03
-placeholder: true
+date: 2021-12-03
 ---
 
-Coming soon!
+Users expect apps to start on slow or flaky network connections, or even when offline.
+They expect the content they've most recently interacted with, such as media tracks or tickets and itineraries, to be available and usable.
+When a request isn't possible, they expect the app to tell them instead of silently failing or crashing. And users wish to do it all quickly. As we can see in this study [Milliseconds make millions](/milliseconds-make-millions/), even a 0.1 second improvement in load times can improve conversion by up to 10%. In summary: users expect PWAs to be reliable and that's why we have service workers.
+
+## Hello service workers
+
+{% Img src="image/RK2djpBgopg9kzCyJbUSjhEGmnw1/iKWO7c2WNobLt30VZx9C.png",
+alt="A service worker as a middleware proxy, running device-side, between your PWA and servers, which includes both your own servers and cross-domain servers.", width="800", height="354" %}
+
+When an app requests a resource covered by the service worker's scope, including when a user is offline, the service worker intercepts the request, acting as a network proxy. It can then decide if it should serve the resource from the cache via the Cache Storage API, from the network as normally would happen without a service worker, or synthesize it from a local algorithm. This lets you provide a similar experience to that provided by a platform app. It can even work entirely off line.
```

Comment:
```suggestion
When an app requests a resource covered by the service worker's scope, including when a user is offline, the service worker intercepts the request, acting as a network proxy. It can then decide if it should serve the resource from the cache via the Cache Storage API, from the network as normally would happen without a service worker, or create it from a local algorithm. This lets you provide a similar experience to that provided by a platform app. It can even work entirely off line.
```